README.md

```diff
@@ -55,17 +57,32 @@ var api *cloudflare.API
 
```

Comment:
Sorry, I only just saw this in (full) preview (doesn't show in the diff): line 56 - `var api *cloudflare.API` - is shadowed by line 60, so you can just remove line 56 for now.